seqr/views/utils/anvil_metadata_utils.py

```diff
@@ -165,7 +165,7 @@ def parse_anvil_metadata(
         airtable_fields: Iterable[str] = None, mme_value: Aggregate = None, include_svs: bool = True,
         variant_json_fields: Iterable[str] = None, variant_attr_fields: Iterable[str] = None, post_process_variant: Callable[[dict, list[dict]], dict] = None,
         include_no_individual_families: bool = False, omit_airtable: bool = False, include_family_name_display: bool = False, include_family_sample_metadata: bool = False,
-        include_discovery_sample_id: bool = False, include_mondo: bool = False, include_parent_mnvs: bool = False,
+        include_discovery_sample_id: bool = False, include_mondo: bool = False, omit_parent_mnvs: bool = False,
```

Comment:
changes the default behavior of this function to include parent MNVs since that is the behavior we want almost everywhere, so the one place we don't want it sets that explicitly